grails-app/migrations/views/product-stockout.sql

```diff
@@ -0,0 +1,35 @@
+CREATE OR REPLACE VIEW product_stockout AS
+(
```

Comment:
LGTM, but the alignment of the brackets in this file is misleading. I thought that this one was not closed.

Reply:
I thought the same thing but this is what IntelliJ does when I run Reformat Code.